packages/website/src/js/fetchGitHubRepoData.js

```diff
@@ -0,0 +1,61 @@
+const githubStarsText = body.querySelector("#github-stars")
+const githubDependentsText = body.querySelector("#github-dependents")
+
+const githubPackageDependentsURL =
+  "https://github.com/TrueFiEng/useDApp/network/dependents?package_id="
+const githubPackageID = "UGFja2FnZS0xODg5OTgyMTY5"
+const proxyURL = "https://api.allorigins.win/raw?url="
+
+const showFetchedStars = async () => {
+  githubStarsText.textContent = "1.2k+"
```

Comment:
```suggestion
  githubStarsText.textContent = "1.2k"
```

We already have the word `Over` so the plus duplicates it.